tests/migrations/0001_initial.py

```diff
@@ -64,4 +64,27 @@ class Migration(migrations.Migration):
             },
             bases=(models.Model,),
         ),
+        migrations.CreateModel(
```

Comment:
Usually I would agree with you, and I would never have done that for the project's main migrations. But those are the migrations for the test fixtures - that database is setup *from scratch* for every test and completely torn down afterward. Because of that, nobody should have any of those objects in their persistent database, so modifying that migration won't/shouldn't impact any users. Having multiple migrations for the test database is, as far as I know, pointless.

Let me know if you still want me to create a new migration instead - I'm more interested in getting this merged.